service/src/test/java/uk/gov/hmcts/reform/fpl/service/HearingBookingServiceTest.java

```diff
@@ -65,6 +67,31 @@ void shouldGetMostUrgentHearingBookingFromACollectionOfHearingBookings() {
         assertThat(sortedHearingBooking.getStartDate()).isEqualTo(TODAYS_DATE);
     }
 
+    @Test
+    void shouldGetHearingBookingWhenKeyMatchesHearingBookingElementUUID() {
+        List<Element<HearingBooking>> hearingBookings = createHearingBookings();
+        HearingBooking hearingBooking = service.getHearingBookingByUUID(hearingBookings, UUIDS[2]);
+        assertThat(hearingBooking.getStartDate()).isEqualTo(TODAYS_DATE);
+    }
+
+    @Test
+    void shouldReturnEmptyHearingBookingWhenKeyDoesNotMatchHearingBookingElementUUID() {
+        List<Element<HearingBooking>> hearingBookings = createHearingBookings();
+        HearingBooking hearingBooking =
+            service.getHearingBookingByUUID(hearingBookings, fromString("b15eb00f-e151-47f2-8e5f-374cc6fc2606"));
+
+        assertThat(hearingBooking).isEqualTo(HearingBooking.builder().build());
+    }
+
+    @Test
+    void shouldGetDynamicHearingBookingElementFromDynamicList() {
+        HearingDateDynamicElement dateDynamicElement =
+            service.getHearingDynamicElement(createHearingBookingDynmaicList());
+
+        assertThat(dateDynamicElement.getId()).isEqualTo(UUIDS[0]);
```

Comment:
I don't like we have got UUIDS[0] and then UUIDS[2] never 1. They probably have got some meaning, do you think it would be beneficial to add additional variable telling the UUID is from earliest hearing? Or something like that? Not clear why this particular one is returned.

Reply:
I completely agree, I would prefer these to be split from the array and have more meaningful names. Perhaps TODAY_PLUS_5 or something similar etc. But i've inherited this code rather than created it. Can we leave this for tech debt?

Reply:
Yes we can, don't need to change right now!